frontend/apps/picasso/defi/polkadot/pallets/Assets/statemine.ts

```diff
@@ -0,0 +1,60 @@
+import { ApiPromise } from "@polkadot/api";
+import { hexToString } from "@polkadot/util";
+import { fromChainIdUnit } from "shared";
+import BigNumber from "bignumber.js";
+
+export type StatemineAssetMetadata = {
+  name: string;
+  decimals: number;
+  symbol: string;
+  deposit: number;
+  id: string;
+  existentialDeposit: BigNumber | null;
+};
+
+type StatemineAssetDetail = {
+  owner: string;
+  issuer: string;
+  freezer: string;
+  supply: number;
+  deposit: number;
+  minBalance: number;
+  isSufficient: true;
+  accounts: number;
+  sufficients: number;
```

Comment:
```suggestion
  sufficient: number;
```
Nit spelling, can we rename this without breaking anything?

Reply:
this is coming from the api. we unfortunately need to keep this.

Reply:
```suggestion
  sufficients: number; // Cspell:ignore sufficients
```
This solves it for the entire file. If we add more repeated use of `sufficients` outside of this file we should add it to the dictionary in the future.